docs/api/assertions/_category_.json

```diff
@@ -1,4 +0,0 @@
-{
-  "label": "Assertions",
```

Comment:
I still believe we need to have a seperate Assertions Section in the API docs to guide for easy reference by users. Writing tests are really composes of:

1. query something
2. do something (action - maybe)
3. assert state

Assertions are fundamental to the validity of the test and should be easy to find - regardless of the implementation details under the hood.

Reply:
@flotwig had a good point in slack earlier:

> So since we've split the "commands" in the docs into "Assertions", "Actions", "Queries", and "Other Commands", I've found it kinda difficult to find the command I'm looking for. I'm sure I'll get the hang of it eventually but it's more difficult than searching through a single a-z list.
> Was there any discussion about keeping them in a single a-z list and using badges to denote the types? like

```
and [assert]
click [action]
debug [query]
eq [query]
type [action]
reload
```

I agree with your point that there's probably some need to have assertions called out right at the top - they're very important. Wondering how we want to structure this section as a whole though, I think breaking up the commands by type may not be quite the right approach.

Maybe we should make a new index page which breaks up commands by type, and then leave the sidebar entirely alphabetical / undifferentiated?

Reply:
That can get us the best of both worlds - users click on API and are presented with a useful index, showing things they need to learn, but then also there's easy access to 'where are the docs on command X' scrolling down an alphabetical list.

Reply:
Based on slack discussions, I've redone the layout of the API section of the docs. Give it a look and let me know what you think.

Reply:
Could you summarize the changes you made to the layout in the PR comment? It's kinda hard to tell what's going on by looking at the files changed

Reply:
I should have done this before sending back to review. Updated PR description with more detail now that the scope has changed.